sample_files/seating_configs/Academy_Hall/Auditorium.json

```diff
@@ -0,0 +1,15 @@
+{
+    "left": {
+	"PURPLE": {
+	    "rows": [1,2,3,4,5,6,7],
+	    "color": "#bb00ff"
+	}
+    },
+        
+    "right": {
+	"ORANGE": {
+	    "rows": [1,2,3,4,5,6,7],
+	    "color": "#ffaa00"
+	}
+    }
+}
```

Comment:
Just a comment: Academy Hall Auditorium is frequently rearranged.  When I had class there last fall, the room had multiple different row/column layouts throughout the semester.  There's nothing we can really do from the Submitty side for this, but I thought it was worth noting.